frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -257,16 +264,23 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
      */
     updateTranslationStatus = (translationId: number, change: ChangeOperation) => {
         const { locale, nextEntity, parameters, pluralForm, router, selectedEntity, dispatch } = this.props;
-        dispatch(history.actions.updateStatus(
-            change,
-            selectedEntity,
-            locale,
-            parameters.resource,
-            pluralForm,
-            translationId,
-            nextEntity,
-            router,
-        ));
```

Comment:
I'd also add a comment here along the lines of:
```
No need to check for unsaved changes in `EditorBase.updateTranslationStatus()`,
because it cannot be triggered for the use case of bug 1508474.
```

Reply:
Done